models/Asset.php

```diff
@@ -290,7 +290,9 @@ public static function getById($id, $force = false)
 
                 $asset->resetDirtyMap();
 
-                \Pimcore\Cache::save($asset, $cacheKey);
+                if (!$force) {
```

Comment:
I wouldn't do this. 
I think it would be better to just set a private flag in CoreHandler as soon as `writeSaveQueue()` is called (reset to `false` when done), which is then checked in `addToSaveQueue()` and ignores queuing items if set to `true`. With that we have covered this topic entirely in the cache where it belongs to.